ui/intermittent-failures/BugDetailsView.jsx

```diff
@@ -71,6 +74,15 @@ const BugDetailsView = (props) => {
     ({ graphOneData, graphTwoData } = calculateMetrics(graphData));
   }
 
+  const updateTable = (state) => {
+    if (state.page + 1 !== page || state.pageSize !== pageSize) {
+      updateState({ page: state.page + 1, pageSize: state.pageSize }, true);
+    } else if (state.sorted.length > 0) {
+      const params = { bug, startday, endday, tree, page, page_size: pageSize, column: state.sorted[0].id, desc: state.sorted[0].desc };
+      getTableData(createApiUrl(defaultState.endpoint, params));
```

Comment:
We don't need to update the `withView` state since `column` and `desc` only applies to bugdetails' table data API and I don't think there's a need to add those to the url querystring.